src/main/kotlin/org/wycliffeassociates/resourcecontainer/IResourceContainerAccessor.kt

```diff
@@ -0,0 +1,11 @@
+package org.wycliffeassociates.resourcecontainer
+
+import java.io.BufferedReader
+import java.io.BufferedWriter
+
+interface IResourceContainerAccessor {
+    fun checkFileExists(filename: String): Boolean
+    fun getReader(filename: String): BufferedReader
```

Comment:
why is a BufferedReader specifically required?

Reply:
Replaced this and BufferedWriters with Reader/Writer in function signatures.